### PR TITLE
Fix imports order to match the new Checstyle rule

### PR DESCRIPTION
**Pull request by ppalaga**

### CODE REVIEW

**Comment by ppalaga** [comment]
@jpkrohling could you please review?

File: sample/src/main/java/org/hawkular/accounts/sample/boundary/SampleEndpoint.java [hunk 1]
```diff
@@ -16,16 +16,7 @@
...
-import org.hawkular.accounts.api.PermissionChecker;
-import org.hawkular.accounts.api.ResourceService;
-import org.hawkular.accounts.api.NamedOperation;
-import org.hawkular.accounts.api.model.Operation;
-import org.hawkular.accounts.api.model.Persona;
-import org.hawkular.accounts.api.model.Resource;
-import org.hawkular.accounts.sample.control.HawkularAccountsSample;
-import org.hawkular.accounts.sample.entity.Sample;
-import org.hawkular.accounts.sample.entity.SampleRequest;
-import org.hawkular.accounts.sample.entity.Sample_;
```
Comment (jpkrohling) [inline sample/src/main/java/org/hawkular/accounts/sample/boundary/SampleEndpoint.java:28]: This one is now missing, causing a compilation problem :-)

Comment (jpkrohling) [inline sample/src/main/java/org/hawkular/accounts/sample/boundary/SampleEndpoint.java:28]: By the way: you can build it locally by using the "sample" and "distribution" profiles: mvn clean install -Psample,distribution .

**Comment by jpkrohling** [comment]
Replaced by #41

**Pull request closed without merging**